base/strings/string.jl

```diff
@@ -59,6 +59,13 @@ String(s::Symbol) = unsafe_string(Cstring(s))
 pointer(s::String) = unsafe_convert(Ptr{UInt8}, s)
 pointer(s::String, i::Integer) = pointer(s)+(i-1)
 
+function unsafe_load(s::String, i::Integer=1)
```

Comment:
This doesn't seem like the right name for this function, as we've been deprecating names of this form. It seems like it should be spelled `@inbounds codeunit(...)` instead, following the example of the array code.